flask_rebar/__init__.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import unicode_literals
```

Comment:
I saw that we only run CI/CD unit tests for python 3.8 and later. I assumed this meant we don't support python 2.0 anymore, but maybe that isnt the case?

or maybe we should release a new major version that drops python 2.0 support?

Reply:
We are currently supporting Python 3.8 through Python 3.11 : https://github.com/plangrid/flask-rebar/blob/master/setup.py#L44-L47

Reply:
I for one would welcome a PR that cleans up all the compatability cruft that we no longer need to maintain 👍

Reply:
I ran across a pre-commit hook that did this automatically, let me see if I can apply that here

Reply:
@airstandley I ran `pyupgrade` on the entire repo to remove pre-3.8 compatibility code